packages/plugin-webpack/plugin.js

```diff
@@ -64,24 +65,25 @@ module.exports = function plugin(config, args) {
         fs.readFileSync(path.join(srcDirectory, config.devOptions.fallback))
       );
 
-      //Find the first local script, assume it's the entrypoint
-      let scriptEl;
-      for (const el of Array.from(
-        dom.window.document.querySelectorAll("script")
-      )) {
-        let src = el.src.trim();
-        if (
-          el.type.trim().toLowerCase() === "module" &&
-          !src.trim().match(/[a-zA-Z+]+:\/\//g)
-        ) {
-          scriptEl = el;
-          break;
+      //Find all local script, use it as the entrypoint
+      const scripts = Array.from(dom.window.document.querySelectorAll("script"))
+        .filter(el => el.type.trim().toLowerCase() === "module")
+        .filter(el => !/^[a-zA-Z]+:\/\//.test(el.src));
+
+      if (scripts.length === 0) {
+        throw new Error("Can't bundle without script tag in html");
         }
+
+      const entries = {};
+      for (const el of scripts) {
+        const src = el.src.trim();
+        const parsedPath = path.parse(src);
+        const name = parsedPath.name;
+        if (entries.name !== undefined) {
+          throw new Error(`Duplictate script with name ${name}`);
```

Comment:
sp: `Duplicate`

Reply:
I like the approach you did here of using the JS filename as the entry name. While collisions can probably be a little confusing here, I think this error message is as good as it can be. I’d love to add a little more info on how this works, maybe on https://www.snowpack.dev/plugins. Because this is an “official” plugin